[All Hosts] (runtimes) mostly links to new runtimes article (#3600)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-overview.md

```

> [!TIP]

> > If your custom function add-in will use a task pane or a ribbon button, in addition to running custom function code, you will need to set up a [shared runtime](../testing/runtimes.md#shared-runtime). To learn more, see [Configure your Office Add-in to use a shared runtime](../develop/configure-your-add-in-to-use-a-shared-runtime.md).

Reviewer: ```suggestion
> If your custom function add-in will use a task pane or a ribbon button, in addition to running custom function code, you'll need to set up a [shared runtime](../testing/runtimes.md#shared-runtime). To learn more, see [Configure your Office Add-in to use a shared runtime](../develop/configure-your-add-in-to-use-a-shared-runtime.md).
```

- Uses **\<ExtensionPoint\>** and **\<Resources\>** elements that are unique to a custom functions manifest. These elements contain the information about the locations of the JavaScript, JSON, and HTML files.
- Specifies which runtime to use for your custom function. We recommend always using a shared runtime unless you have a specific need for another runtime, because a shared runtime allows for the sharing of data between functions and the task pane.

> If you are using the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) to create files, we recommend adjusting your manifest to use a shared runtime, as this is not the default for these files. To change your manifest, follow the instructions in [Configure your Excel add-in to use a shared runtime](../develop/configure-your-add-in-to-use-a-shared-runtime.md).

Reviewer: ```suggestion
If you're using the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) to create files, we recommend adjusting your manifest to use a shared runtime, as this is not the default for these files. To change your manifest, follow the instructions in [Configure your Excel add-in to use a shared runtime](../develop/configure-your-add-in-to-use-a-shared-runtime.md).
```
